### DISCUSSION TITLE
Does Encrypted Stage listener support IV and key generated by Openssl in hex ?

### DISCUSSION THREAD

**Discussion by Krishna8009** [BishopFox/sliver#1081] [Q&A]
Openssl generates key and IV as hex of 32 bytes for AES-CBC-128. I am using command `openssl enc -aes-128-cbc  -p -k somesecret -nosalt`. The Hex IV will convert to 16 bytes in decimal notation as per my understanding.
 However in documentation i see  a decimal notation IV of 16 bytes. Please can you tell me what should i do if i want to use openssl generated IV and Key or if i can generate keys with some alternative tools so that it works with the stage listener ?

**Comment by aarjn** [comment]
If you want to use the key and IV generated by OpenSSL for AES-CBC-128 encryption, you can use the -K and -iv options of the openssl enc command to specify the key and IV values in hexadecimal notation.

**Reply by D3m0cr1tusGR** [reply]
Could you please provide an example of the command to generate the key and iv with openssl. Sliver crashes when I am trying to use the  generated key and iv.
